akka-persistence/src/main/scala/akka/persistence/fsm/PersistentFSM.scala

```diff
@@ -54,6 +54,19 @@ trait PersistentFSM[S <: FSMState, D, E] extends PersistentActor with Persistent
   private var currentStateTimeout: Option[FiniteDuration] = None
 
   /**
+   * "akka.persistence.fsm.snapshot-after"
+   */
+  private val snapshotAfter: Option[Int] = {
+    val config = context.system.settings.config
+    val key = "akka.persistence.fsm.snapshot-after"
+
+    config.getString(key).toLowerCase match {
+      case "off" ⇒ None
+      case _     ⇒ Some(config.getInt(key))
+    }
+  }
```

Comment:
Thanks for the feedback.

(Just in case, I tried `lazy val snapshotAfter ... `, but encountered `AbstractMethodError` too.)

So it seems Akka Extension is the only possibility.
Let me spend maybe few weeks(?) on it as I'm new to Akka Extension.